protocol/scripts/deploys/sepolia/templegold/99-post-create-spice-verify.ts

```diff
@@ -0,0 +1,68 @@
+import '@nomiclabs/hardhat-ethers';
+import '@nomiclabs/hardhat-etherscan';
+import { ethers, run } from 'hardhat';
+import {
+  ensureExpectedEnvvars,
+  mine
+} from '../../helpers';
+import {
+    getDeployedTempleGoldContracts,
+    connectToContracts,
+    ContractInstances
+} from '../../mainnet/templegold/contract-addresses';
+
+async function main() {
+    ensureExpectedEnvvars();
+    const [owner] = await ethers.getSigners();
+    const ownerAddress = await owner.getAddress();
+    const TEMPLEGOLD_ADDRESSES = getDeployedTempleGoldContracts();
+
+    const TEMPLE_GOLD_INSTANCES = connectToContracts(owner);
+    const name = ""; //"TGLD_TOKENNAME_SPICE";
+    const spiceToken = "";
+
+    await mine(TEMPLE_GOLD_INSTANCES.TEMPLE_GOLD.SPICE_AUCTION_FACTORY.createAuction(spiceToken, name));
+    const spiceAuction = await TEMPLE_GOLD_INSTANCES.TEMPLE_GOLD.SPICE_AUCTION_FACTORY.findAuctionForSpiceToken(spiceToken);
+    
+    await run("verify:verify", {
```

Comment:
oh that's cool - didn't know that was possible!

Reply:
Yes same params, unless you need a param not known at time of running deploy script. This will be a good addition for future chore to perhaps move away from the deploymentArgs

Reply:
If it works - eg sometimes the verification service is down on some new chains, so need to do later

Reply:
could do a try once, and if error, write command to deployArgs